src/views/TwabRewards/hooks/useAccountsPromotions.ts

```diff
@@ -0,0 +1,62 @@
+import gql from 'graphql-tag'
+import { useQuery } from 'react-query'
+import { NO_REFETCH } from '@pooltogether/hooks/dist/constants'
+
+import { useTwabRewardsSubgraphClient } from '@twabRewards/hooks/useTwabRewardsSubgraphClient'
+
+/**
+ *
+ * @param chainId
+ * @param account
+ * @returns Promotion[]
+ */
+export const useAccountsPromotions = (chainId: number, account: string) => {
+  return useQuery(
+    ['useAccountPromotions', account, chainId],
+    async () => getAccountsPromotions(chainId, account),
+    {
+      ...NO_REFETCH,
+      enabled: !!account
+    }
+  )
+}
+
+export const getAccountsPromotions = async (chainId, account) => {
+  const query = promotionsQuery()
+  const variables = {
+    accountAddress: account.toLowerCase()
+  }
+
+  const client = useTwabRewardsSubgraphClient(chainId)
+
+  return client.request(query, variables).catch((e) => {
+    console.error(e.message)
+    return null
+  })
+}
+
+const promotionFragment = gql`
```

Comment:
Is there any benefit to making this a fragment vs just inlining it into the query?